src/plugins/intel_cpu/tests/functional/shared_tests_instances/snippets/transpose_matmul.cpp

```diff
@@ -11,7 +11,7 @@ namespace ov {
 namespace test {
 namespace snippets {
 
-#define STATIC_SHAPES(...) static_shapes_to_test_representation(std::vector<std::vector<ov::Shape>>{__VA_ARGS__})
+#define SNIPPETS_TESTS_STATIC_SHAPES(...) static_shapes_to_test_representation(std::vector<std::vector<ov::Shape>>{__VA_ARGS__})
```

Comment:
Why can't we use the one from utils.hpp here?

Reply:
We can do it, for sure. I think that I just skipped this file when I refactored the tests. Reused from `utils.hpp`. Thanks!